### PR TITLE
Update portuguese translation

### PR DESCRIPTION
**Pull request by gustavobite**
Update portuguese translation

### CODE REVIEW

**Comment by hzsweers** [comment]
Thanks! Do you have another Portuguese speaker that could double check these? Just to sanity check with a second set of eyes (spelling, grammar, etc)

**Comment by gustavobite** [comment]
You are welcome. Sorry, but I don't have another portuguese speaker for check these. Thanks!

**Comment by hzsweers** [comment]
@fortes could you double check these?

**Comment by gustavobite** [comment]
@feliperuhland could you check these?

File: app/res/values-pt/strings.xml [hunk 1]
```diff
@@ -271,5 +271,22 @@
     <string name="preview">Prever</string>
     <string name="show_raw_markdown">Mostrar markdown puro</string>
     <string name="render_markdown">Renderizar markdown</string>
+    <string name="are_you_sure">Você está absolutamente certo?</string>
+    <string name="code_authentication_failed">Falha de autenticação! Código de verificação incorreto</string>
+    <string name="confirm_comment_delete_message">Tem certeza de que deseja excluir este comentário?</string>
+    <string name="confirm_comment_delete_title">Remover Comentário</string>
+    <string name="copy_hash">Copiar hash</string>
+    <string name="delete_cap">EXCLUIR</string>
```
Comment (fortes) [inline app/res/values-pt/strings.xml]: Might be better to use be *apagar* or *deletar* in these `delete_*` to better match the English version. Especially in the repository case, *excluir* might read like hiding/omission, which would give the wrong impression given that it is undoable.

**Comment by fortes** [comment]
A note about the `delete_*` translations, otherwise :+1:

**Comment by gustavobite** [comment]
Good observation @fortes . I did the suggested changes. You can see here: http://goo.gl/tMJy8L

**Comment by feliperuhland** [comment]
Hey @gustavobite @fortes.

Personally I don't like the word **apagar**, I find most interesting **remover** or **excluir**.
But, @fortes has a good argument, independent of the word, it must be the same in all posts.

**Comment by gustavobite** [comment]
@fortes  @feliperuhland 

The word varies by context, in my opinion. 
For example, the right for me in this sentence 'Deleting failed' is 'Exclusão falhou'.

Thank your opinions and help.

**Comment by feliperuhland** [comment]
Like I said, it's a personal opinion. 
Anyway, your translation is fine and should be merged.

**Comment by fortes** [comment]
**Remover** works for me as well, but if others feel **Excluir** is better, then so be it.

@gustavobite I would just make sure the word is consistent, your last checkin still has **excluir** in `delete_successful`

**Comment by hzsweers** [comment]
Let's go with consistency, both in the translations and relative to other languages. If *remover* makes more sense, let's go with that. And also update [this line](https://github.com/gustavobite/android/commit/8d14378d9e03da1d345366990632a9e2aea9ad48#diff-a47202a66991135560d1272be0208f01R281) to match

**Comment by hzsweers** [comment]
Also thanks to everyone for the discussion, glad to have good translations :)

**Comment by fadils** [comment]
I believe this is ready to merge, right?

**Comment by feliperuhland** [comment]
@fadils Despite any disagreement, the translation is good and is ready to merge.

**Comment by hzsweers** [comment]
Thanks everyone!

**Pull request merged**